tensorflow/tools/benchmark/benchmark_model.cc

```diff
@@ -209,7 +209,8 @@ Status CalculateFlops(const GraphDef& graph,
 
 Status RunBenchmark(const std::vector<InputLayerInfo>& inputs,
```

Comment:
Maybe add comment that when step_stats is requested, the inference time may be pessimistic?